fanficfare/adapters/adapter_literotica.py

```diff
@@ -220,19 +220,18 @@ def extractChapterUrlsAndMetadata(self):
         else:
             ## Multi-chapter stories.  AKA multi-part 'Story Series'.
             bn_antags = soup.select('div#tabpanel-info p.bn_an')
-            # logger.debug(bn_antags)
-            if bn_antags:
-                dates = []
-                for datetag in bn_antags[:2]:
-                    datetxt = stripHTML(datetag)
-                    # remove 'Started:' 'Updated:'
-                    # Assume can't use 'Started:' 'Updated:' (vs [0] or [1]) because of lang localization
-                    datetxt = datetxt[datetxt.index(':')+1:]
-                    dates.append(datetxt)
-                # logger.debug(dates)
-                self.story.setMetadata('datePublished', makeDate(dates[0], self.dateformat))
-                self.story.setMetadata('dateUpdated', makeDate(dates[1], self.dateformat))
 
+            # Custom code to set pub and update dates to approved dates on oldest and newest chapters respectively
+            date = re.findall(r'"date_approve":"(\d\d/\d\d/\d\d\d\d)"',data)
```

Comment:
This is taking from the JSON in the javascript.  There's already code for reading and parsing the json.  Maybe try using that?  

Search the file for `json`, the `averrating` is calculated using it.  Don't need to parse the json again, but the exception catching should be changed.